src/factories/CommandFactory.ts

```diff
@@ -20,17 +22,27 @@ class CommandFactory {
 			const aliases = new Command().getAliases();
 
 			aliases.forEach((alias: string) => {
-				this.commands[alias] = () => new Command();
+				this.commandAliases[alias] = () => new Command();
 			});
 		});
 	}
 
-	commandExists(command: string): boolean {
-		return typeof this.commands[command.toLowerCase()] !== "undefined";
+	commandOrAliasExists(command: string): boolean {
+		const commandsAndAliases = {...this.commandAliases, ...this.commands};
+
+		return typeof commandsAndAliases[command.toLowerCase()] !== "undefined";
```

Comment:
Why check `typeof something !== "undefined"` instead of just `something !== undefined`?